varcode/vcf.py

```diff
@@ -65,19 +65,21 @@ def load_vcf(
 
     ensembl = EnsemblRelease(release=ensembl_release)
 
-    # We ignore "no-call" variants, i.e. those where X.ALT = [None].
-    variants = [
-        Variant(
-            contig=x.CHROM,
-            pos=x.POS,
-            ref=x.REF,
-            alt=x.ALT[0].sequence,
-            info=x.INFO,
-            ensembl=ensembl)
-        for x in vcf_reader
-        if x.ALT[0] and (not only_passing or not x.FILTER or x.FILTER == "PASS")
-    ]
-
+    variants = []
+    for record in vcf_reader:
```

Comment:
I suppose this has always been the case for varcode, but this just made me think about it: we're throwing away the genotype information. Even for a regular non-multiallelic variant, I don't think I have a way to see if it was called a het or hom. Would it make sense to retain that in the variant? Or we could potentially address that in a later PR.